src/js/providers/html5.js

```diff
@@ -514,6 +514,10 @@ define([
                 }
             } else {
                 _delayedSeek = seekPos;
+                // Some browsers won't fire canplay event in a paused state (Firefox)
+                if (_videotag.paused) {
```

Comment:
Branching on user agent is bad practice. In my testing this change resulted in quality change and midroll resumes that performed better in Chrome and iOS because it no longer depends on the round about "buffer full" event triggering play for selling to occur.